lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,13 +552,15 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.verbose("The network request completed successfully, but no metadata was returned.");
                 return null;
             }
             match = Authority.getCloudDiscoveryMetadataFromNetworkResponse(
                 metadata,
                 this.hostnameAndPort
             );
         } catch (e) {
+            this.logger.verbose("There was a network error while attempting to get the cloud discovery metadata.");
             return null;
         }
 
```

Comment:
Please add a `log.Warning` in the `if (!match)` statement, as this will tell us if instance discoevry failed.